diff --git a/contrib/linux/meson.build b/contrib/linux/meson.build
index 1ecd70104..c76d7022b 100644
--- a/contrib/linux/meson.build
+++ b/contrib/linux/meson.build
@@ -1,3 +1 @@
 install_data('dosbox-staging.desktop', install_dir: data_dir / 'applications')
-
-install_data('dosbox-staging.metainfo.xml', install_dir: data_dir / 'metainfo')
diff --git a/meson.build b/meson.build
index a31379aac..0978c3a92 100644
--- a/meson.build
+++ b/meson.build
@@ -38,18 +38,7 @@ licenses_dir = data_dir / 'licenses' / meson.project_name()
 doc_dir = data_dir / 'doc' / meson.project_name()
 
 install_man('docs/dosbox.1')
-# Bundle licenses, but skip the ones that are not relevant for
-# binary distribution or allow us to not distribute the license text.
-install_data(
-    'LICENSE',
-    'licenses/BSD-2-Clause.txt',
-    'licenses/BSD-3-Clause.txt',
-    'licenses/GPL-2.0.txt',
-    'licenses/LGPL-2.1.txt',
-    'licenses/MIT.txt',
-    'licenses/Zlib.txt',
-    install_dir: licenses_dir,
-)
+install_data('LICENSE', install_dir: doc_dir)
 install_data('AUTHORS', 'README', 'THANKS', install_dir: doc_dir)
 
 subdir('contrib/linux')
